types/mongoose/index.d.ts

```diff
@@ -1048,9 +1048,9 @@ declare module "mongoose" {
   /*
    * Intellisense for Schema definitions
    */
-  interface SchemaDefinition {
-    [path: string]: SchemaTypeOpts<any> | Schema | SchemaType;
-  }
+  type SchemaDefinition<T> = {
```

Comment:
Would you consider making it backwards compatible by adding a default, eg
```suggestion
  type SchemaDefinition<T = {[x: string]: unknown}> = {
```
?